packages/cascara/src/ui/Table/Table.js

```diff
@@ -0,0 +1,35 @@
+import React, { useContext } from 'react';
+
+import TableContext from './context/TableContext';
+
+import TableHeader from './TableHeader';
+import TableBody from './TableBody';
+
+const Table = () => {
+  const {
+    dataConfig: { actions, bulkActions, display },
+  } = useContext(TableContext);
```

Comment:
It looks like right now all the data is getting defined directly in the table context? How are end users getting data into that context? I don't think we want to export contexts from Cascara. We should probably be exposing props on this Table and then passing that into context within this component. Unless this Table is not actually the official component where that happens?

Reply:
You are right, Table is wrapped by the TableContext. The index in `packages/cascara/src/ui/Table` exports the